root/tree/index/runindex64.C

```diff
@@ -25,20 +27,22 @@ int runindex64(){
   tree->Branch("run", &run, "run/l");
   tree->Branch("event", &event, "event/l");
 
-  ULong64_t events[] = { 1,2,3, bigval, biguval, 5 };
-  run = 5;
+  ULong64_t   runs[] = { 8,5,5,5,      0,       0,  5,      4, 6, biguval,          1, maxMajor, maxMajor+1};
+  ULong64_t events[] = { 0,1,3,2, bigval, biguval,  5, bigval, 3,  bigval, maxMinor+1, maxMinor, 0};
```

Comment:
These should become `std::array<ULong64_t, 13>`

Reply:
thanks! But one advantage of this formulation over std::array is that you don't have to count nr of elements (13), it's just auto-initialized. see https://comp.lang.cpp.moderated.narkive.com/ZMoUm1AY/std-array-if-only-it-knew-its-size